tools-image/build-arm-image.sh

```diff
@@ -328,6 +333,8 @@ if [ -n "$EFI" ] && [  -n "$efi_dir" ]; then
   cp -rfv $efi_dir/* $EFI
 fi
 
+partprobe
```

Comment:
not sure if this is needed, I remember I got some error reading partitions at some point. It doesn't hurt though.